lte/gateway/c/sctpd/src/sctpd.cpp

```diff
@@ -75,11 +83,41 @@ int signalHandler(int* end, std::unique_ptr<Server>& server,
   return 0;
 }
 
+static magma::mconfig::SctpD load_sctpd_mconfig() {
+  magma::mconfig::SctpD mconfig;
+  if (!magma::load_service_mconfig_from_file(SCTPD_SERVICE, &mconfig)) {
+    mconfig.set_log_level(magma::orc8r::LogLevel::INFO);
+  }
+  return mconfig;
+}
+
+static magma::mconfig::SharedMconfig load_shared_mconfig() {
+  magma::mconfig::SharedMconfig mconfig;
+  magma::load_service_mconfig_from_file(SHARED_MCONFIG, &mconfig);
+  return mconfig;
+}
+
+static uint32_t get_log_verbosity(magma::mconfig::SctpD mconfig) {
```

Comment:
Might be good to place this function in the common mconfig library so that it can be used by other services as well?

Reply:
Done in #10969, will rebase and do it for sctpd when one of the commits is merged.